src/huggingface_hub/commands/delete_cache.py

```diff
@@ -77,24 +76,6 @@
     _inquirer_py_available = False
 
 
-def require_inquirer_py(fn: Callable) -> Callable:
```

Comment:
(changes not needed in the end if we make inquire_py optional in auth switch, sorry about that :see_no_evil: )